src/openforms/submissions/admin.py

```diff
@@ -230,6 +230,7 @@ class SubmissionAdmin(admin.ModelAdmin):
         "payment_complete_confirmation_email_sent",
         "get_price",
         "get_payment_required",
+        "object_reference",
```

Comment:
let's leave it editable instead of making it read-only, all these readonly fields are making problem solving hard for Laurens and me